### ISSUE TITLE
Doesn't work in Ubuntu 17.04

### ISSUE THREAD

**Issue by octoxan** [scotch-io/scotch-box#301]
On a fresh install of Ubuntu 17.04, I installed Virtual Box, Vagrant, and then Scotchbox (nothing else at all yet) and while I can vagrant ssh into the box, trying to go to 192.168.33.10 just times out.

Any ideas?

**Comment by whatnickcodes** [comment]
Sounds like a Vagrant / Virtualization / Virtual Box thing. If you can get any other base box working, this one would work too

**Comment by octoxan** [comment]
Forgot to comment back here.

The issue is with Ubuntu 17.04 not including the package 'net-tools' any more.

After installing net-tools I also had to edit my local Ubuntu's hosts file to remove the first line:

127.0.0.1 my-pc-name

Then it all works. Tested this with a few different Ubuntu 17.04 installs now and all of them can't view the vagrant websites until net-tools is installed and that line of the hosts file is removed.
